example/dev-proxy/README.md

```diff
@@ -0,0 +1,32 @@
+# DEV PROXY
+
+This script creates a proxy by simulating what io-backend does for adding information relating to the authenticated user.
+
+# How to use
+
+1. [optional] Use pyenv to manage python version and create a virtual environment
```

Comment:
The only reason is that it was much easier to make it in python than typescript (a lot of it was found already done). I will still try to convert it to TS with an upcoming PR since it's just an additional utility but can also be unused.

Reply:
https://chat.openai.com/share/2e604e28-51df-478f-b625-650c79fac777

Reply:
We tried [here](https://github.com/pagopa/io-react-native-wallet/pull/80), but obviously it doesn't work. It always returns `500`. It should be investigated better and perhaps the code written correctly. I would leave this in python for now since it's just a utility that doesn't have to be used.